docs/configuration.md

```diff
@@ -202,3 +207,20 @@ An example of a batch script:
     wallet close
     exit
     ```
+
+#### Transaction Author Agreement
+CLI uses session-based approach to work with Transaction Author Agreement.
+
+#### Workflow
+1. On CLI start: Pass a config JSON file containing a label of mechanism how a user is going to accept a transaction author agreement.
+`indy-cli --config <path-to-config-json-file>` where config looks like:
+```
+{
+  "taaAcceptanceMechanism": "Click Agreement",
+  .....
+}
+```
+The list of available acceptance mechanisms can be received by sending `get_acceptance_mechanisms` request to ledger.
+1. On `pool connect` command execution: User will be asked if he would like to accept TAA.
```

Comment:
I would add a clarification comment that asking for TAA on pool_connect is just CL-specific behavior. Indy checks for TAA on write requests only.